app/assets/stylesheets/application.scss

```diff
@@ -1,46 +1,67 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+$light-blue: #C4EAFF;
+$medium-blue: #2CA8DD;
+$dark-blue: #000566;
+$white: #FFFFFF;
+$dark-green: #8c8b8b;
+$light-green: #39B348;
+
 body{
-  color: #000566;
-  background-color: #C4EAFF;
+  color: $dark-blue;
+  background-color: $light-blue;
 }
 
 h1{
   text-align: center;
   text-transform: uppercase;
-  text-shadow: 1px 1px 0px #FFF;
+  text-shadow: 1px 1px 0px $white;
 }
+
 hr.shadow{
   height: 10px;
   border: 0;
-  box-shadow: 0 10px 10px -10px #8c8b8b inset;
+  box-shadow: 0 10px 10px -10px $dark-green inset;
+}
+
+.view-container{
+  padding-left: 2em;
```

Comment:
personally prefer using `em`s for font related spacing, and `px`s or `%`s for positioning